docs/articles/new-expensify/expensify-card/View-and-Reconcile-Expensify-Card-Expenses.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Filter and Reconcile the Expensify Cards
+description: Learn how to reconcile Expensify Card transactions using filters and exports in New Expensify, with tips for both card and payment reconciliation.
+keywords: [New Expensify, Expensify Card reconciliation, card filters, export transactions, payment reconciliation, report filters, posted date, settlement matching]
+---
+<div id="new-expensify" markdown="1">
+
+Once employees use the Expensify Card, you can easily view and reconcile their transactions from the Reports page in New Expensify.
+
+
+# View Card Transactions by Employee or Company
+
+To view card transactions for a specific employee or for the entire company:
+
+1. Go to the **Reports** page.
+2. Click **Filters** in the top-right corner.
+3. Click **Card** to filter cards:
+   - Select a feed from **Card feeds** to view expenses from all cards in the feed. 
+   - Select a card from **Individual cards* to view expenses from that card. 
```

Comment:
One `*` is missing to close the bold section

Reply:
![image](https://github.com/user-attachments/assets/02641d70-f227-4e3b-8668-82eb61990c7a)